api/app/signals/apps/feedback/models.py

```diff
@@ -0,0 +1,51 @@
+import secrets
+import uuid
+from datetime import timedelta
+
+from django.contrib.gis.db import models
+from django.utils import timezone
+
+from signals.apps.feedback.managers import FeedbackManager
+from signals.apps.signals.models import Signal
+
+FEEDBACK_EXPECTED_WITHIN_N_DAYS = 14  # move to general settings file
```

Comment:
Already a comment on this line, move to general settings file. 

Or we should make this configurable in the admin. If we do that it is possible to change these settings without deploying a code change. We can make some sort of SingletonModel, for example you can do this with django-solo see https://github.com/lazybird/django-solo

Reply:
cool suggestion, but part of larger conversation